### DISCUSSION TITLE
Open ScaffoldPage on button click

### DISCUSSION THREAD

**Discussion by huntergps** [bdlukaa/fluent_ui#685] [Q&A]
How can I open a page using a button:
I have a list of sales, and I want to click on a button to open a ScaffoldPage with the detailed information of the sale. Can you help me with an example.
Thank you

**Comment by bdlukaa** [comment]
Do you have any design in mind? You can use the navigator to push a new route with the info you want. You can also use `showDialog` to display an alert with the info of the sale. You can also show a bottom sheet with the info of the sale

**Reply by huntergps** [reply]
Thanks to the answer, the idea is to pass a route like this: /sales/sale/1, to show in the body a form to edit the sale information. If you can pass the sample with the browser to prompt a new route, I would be very grateful.

**Reply by huntergps** [reply]
The next screns try to explan the idea:

![image](https://user-images.githubusercontent.com/955008/212790353-d1647c75-70cf-4818-9a77-9587ece8ca12.png)
![image](https://user-images.githubusercontent.com/955008/212790375-ca22875c-326b-4a51-b099-b16943561953.png)
![image](https://user-images.githubusercontent.com/955008/212790702-afe46fdb-a3a2-47f4-b6e8-fb6552c3031d.png)

**Reply by bdlukaa** [reply]
Inner navigation is currently not built in (See #226). But this can be easily implemented by changing the content of the body.

**Reply by Raafa19** [reply]
Hi, sorry to reply a year old discussion, but it's inner navigation (similar to the example before) still not built in?
I try changing the body dynamically using a `PageView` , but it kinda gets difficult to add multiple bodies, especially if the next body depends of the previous.

**Reply by bdlukaa** [reply]
@Raafa19 you can implement inner navigation using Flutter's go_router package. This is beyond the goal of the fluent_ui package.
